sections/discussion/discussion-chapter.tex

```diff
@@ -138,6 +139,9 @@ \subsubsection{Timestamp as a context variable}
 
 \subsubsection{The performance of graph-based methods} 
 The results were shortly examined in \autoref{subsec:resultsofexperiment}, where it was noted that the graph-based methods generally did not perform well.
+It is important to note that the graph-based methods focus on top-$N$ recommendations, to which they utilize implicit ratings.
+This may give them a disadvantage compared to rating prediction methods, since these use explicit ratings meaning that have more available information that can help them order the recommendations for the user.
```

Comment:
```suggestion
This may give them a disadvantage compared to rating prediction methods, since these use explicit ratings meaning they have more available information that can help them order the recommendations for the user.
```